Packs/GoogleChatViaWebhook/Integrations/GoogleChatViaWebhook/GoogleChatViaWebhook.yml

```diff
@@ -76,33 +62,17 @@ script:
     description: Send a customizable card to Google Chat Space via Incoming Webhook.
     name: send-google-chat-custom-card
     outputs:
-    - contextPath: GoogleChatWebhook.CustomCard.Cards
```

Comment:
please do not remove the GoogleChatWebhook.CustomCard.Cards output as we do receive it in the response (it is under cardsV2 in the response and then being handled in line 162)
thanks!

Reply:
Done